README.md

```diff
@@ -60,15 +60,15 @@ Enforce correctness and best practices with the native [Spectral](https://stopli
 
 Clicking on errors or warnings also brings you to exactly where they are located in the document, making it easy to fix errors at the source.
 
-To find out more about Spectral and how validations can improve your API design workflow, see [here](./docs/Design-and-Modeling/validation-style-guide.md).
+To find out more about Spectral and how validations can improve your API design workflow, see [here](./docs/Design-and-Modeling/08-validation-style-guide.md).
 
 ## Built-in Mocking
 
 When running locally, Studio will automatically start a local [Prism mock server](https://stoplight.io/prism/) for every API defined in your project, and keep that mock server up to date as you change your designs.
 
 ![Mocking allows you to quickly test the look and feel of your API before jumping into the code](assets/images/studio-mocking.png)
 
-To find out more about Prism and how mocking can be used to streamline your API development process, see [here](https://stoplight.io/p/docs/gh/stoplightio/studio/docs/Design-and-Modeling/mock-servers.md).
```

Comment:
I've changed this link to a local file ref link, because we had both types and this one seems to be more useful as it will flag up errors in our link checker when/if we rename things.

Reply:
💯 any links to files in this repo should all be relative.